pages/index.js

```diff
@@ -1,11 +1,30 @@
 import MainColor from "@/components/MainColor";
+import Link from "next/link";
+import styled from "styled-components";
 
-export default function HomePage() {
+export const StyledLink = styled(Link)`
+  padding: 15px 40px;
+  border: none;
+  border-radius: 100px;
+  text-decoration: none;
+  height: 70px;
+  width: 250px;
+  font-size: 20px;
+  background-color: #bcbdbc;
+`;
+
+const StyledContainer = styled.div`
```

Comment:
you could rename this to "main" or something that tells the reader something about this section/div. I learned that styledContainer or StyledLink,... is not explicit enough

Reply:
Your're right. Fixed 👍